### PR TITLE
[show]: split show interfaces commands into separate file

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- What I did**
[show]: split show interfaces commands into separate file

add extensive test coverage for show interfaces commands.

**- How I did it**

**- How to verify it**
unit test and on box test

unit test coverage

```
show/interfaces.py                           223     73    67%
```

a few things not covered.

show breakout (need to check with @zxu why it is not covered before)
show interfaces lpmode (i think this should be add by part of platform efforts)
show interfaces counters ... (this has never been added, need new efforts to add this)

all others commands are almost 100% covered by the unit tests.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -95,166 +96,6 @@ def run_command(command, display_cmd=False, return_cmd=False):
 # Global class instance for SONiC interface name to alias conversion
 iface_alias_converter = clicommon.InterfaceAliasConverter()
 
-def print_output_in_alias_mode(output, index):
```
Comment (lguohan) [inline show/main.py:98]: moved to cli.py

File: show/main.py [hunk 2]
```diff
@@ -95,166 +96,6 @@ def run_command(command, display_cmd=False, return_cmd=False):
...
-    if alias_name:
-        if len(alias_name) < iface_alias_converter.alias_max_length:
-            alias_name = alias_name.rjust(
-                                iface_alias_converter.alias_max_length)
-        output = output.replace(interface_name, alias_name, 1)
-
-    click.echo(output.rstrip('\n'))
-
-
-def run_command_in_alias_mode(command):
```
Comment (lguohan) [inline show/main.py:132]: moved to cli.py

File: scripts/intfstat [hunk 3]
```diff
@@ -17,7 +17,7 @@ import time
 
 # mock the redis for unit test purposes #
 try:
-    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
```
Comment (jleveque) [inline scripts/intfstat:20]: What does "2" represent, and how does it differ from "1"?

Comment (lguohan) [reply scripts/intfstat:20]: 1 means skip running the command, 2 means you still run the command, but inside the command, you do db mock. I can add some doc in the next pr to document this behavior.

Comment (jleveque) [reply scripts/intfstat:20]: > I can add some doc in the next pr to document this behavior.

Please do.

**Comment by lguohan** [comment]
retest this please

**Pull request merged**